dataland-frontend/src/components/forms/parts/fields/FormFieldProps.ts

```diff
@@ -81,3 +86,34 @@ export const MultiSelectFormProps = Object.assign(deepCopyObject(FormFieldPropsW
     required: false,
   },
 }) as Readonly<ComponentPropsOptions>;
+
+export const DataPointFormFieldProps = Object.assign(deepCopyObject(YesNoFormFieldProps as ObjectType), {
+  dataTest: {
+    type: String,
+  },
+  placeholder: {
+    type: String,
+  },
+  evidenceDesired: {
+    type: Boolean,
+  },
+}) as Readonly<ComponentPropsOptions>;
+
+export const UploadReportsProps = Object.assign(deepCopyObject(YesNoFormFieldProps as ObjectType), {
+  referencedReportsForPrefill: {
+    type: Object as () => { [key: string]: CompanyReport },
+  },
+  isEuTaxonomy: {
+    type: Boolean,
+    default: false,
+  },
+  dataTest: {
+    type: String,
+  },
+  placeholder: {
+    type: String,
+  },
+  options: {
+    type: Array,
+  },
+}) as Readonly<ComponentPropsOptions>;
```

Comment:
I don't understand why these props are handled here.  This file is meant for props that repeat themselves throughout the code.
But `UploadReportProps` is only used once in `UploadReports.vue`.
So why not just paste them there?

Or why not include the `YesNoFormFieldProps` in `UploadReports.vue`, and then write the additional props underneath?

Reply:
When adding components dynamically using <component v-for""> and adding props that are needed, we get 200+ warnings in the console about inappropriate props. This is a workaround solution until we can further discuss how to better render components dynamically.
Dynamic rendering of components is also present in EU Taxonomy, and we got lucky there that no warnings are displayed.. for some reason. However, theoretically, the same "technical" problem is present there.

Reply:
Ok but "UploadReports.vue" is only renedered once dynamically in the whole frontend code (`referencedReports` field for sfdr).
So I have a feeling that setting the available props directly in the "UploadReports.vue" component instead of setting them here and importing them in the vue-component can make that much of a difference in number of warnings?
What do you think.

(PS: I don't think that we have dynamic rendering in Eu Taxonomy frontend code. It is the only framework, where all upload elements are "hardcoded" into the component.)

Reply:
I have renamed ist to @referenceUpdated.

Reply:
@stewal19 I think your comment is probably about another conversation, right?

Reply:
Yes, i somehow posted it into the wrong conversation

Reply:
Ok then this is still open to be addressed.

Reply:
_referencedReportsForPrefill_  is a good example because we don't broadcast this value widely by generating components, we only make it available via provide so it shouldn't be part of other components except those and take it via inject value. I will change it.

Reply:
👍